From 0eceeecf20541e11cc1cc1bef750003309c41eac Mon Sep 17 00:00:00 2001
From: Sebastian Wagner <sebix@sebix.at>
Date: Mon, 28 Mar 2022 11:46:50 +0200
Subject: [PATCH] setup: fix version number

minor change to fix the version number in setup.py. that mistake leads
to wrong version numbers on the local installation
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: cookiecutter-2.0.2/setup.py
===================================================================
--- cookiecutter-2.0.2.orig/setup.py
+++ cookiecutter-2.0.2/setup.py
@@ -2,7 +2,7 @@
 """cookiecutter distutils configuration."""
 from setuptools import setup
 
-version = "2.0.0"
+version = "2.0.2"
 
 with open('README.md', encoding='utf-8') as readme_file:
     readme = readme_file.read()
@@ -10,7 +10,7 @@ with open('README.md', encoding='utf-8')
 requirements = [
     'binaryornot>=0.4.4',
     'Jinja2>=2.7,<4.0.0',
-    'click>=7.0,<8.0.0',
+    'click>=7.0,<9.0.0',
     'pyyaml>=5.3.1',
     'jinja2-time>=0.2.0',
     'python-slugify>=4.0.0',
